demos/GMW_demo/test.js

```diff
@@ -0,0 +1,132 @@
+// Chai
+var assert = require('chai').assert;
+var mpc = require('./mpc.js');
+
+// Generic Testing Parameters
+var showProgress = true;
+var parallelismDegree = 4; // Max number of test cases running in parallel
+var Zp = 13;
+var n = 200;
+var party_count = 4;
```

Comment:
I see why the mpc.js file has a shares[4] usage based on this. Try and make the demo work generically both for the tests and when run manually, so if someone runs the demo with <4 parties it should still work.